test/unit/liquidity_pool_endpoints_test.js

```diff
@@ -141,4 +141,492 @@ describe('/liquidity_pools tests', function() {
       });
     });
   });
+
+  describe('querying a specific pool', function() {
+    const lpId = "ae44a51f6191ce24414fbd1326e93ccb0ae656f07fc1e37602b11d0802f74b9a";
+
+    const poolOpsResponse = {
+      "_links": {
+        "self": {
+          "href": "https://private-33c60-amm3.apiary-mock.com/liquidity_pools/1/operations?cursor=113725249324879873&limit=10&order=asc"
+        },
+        "next": {
+          "href": "https://private-33c60-amm3.apiary-mock.com/liquidity_pools/1/operations?cursor=113725249324879873&limit=10&order=asc"
+        },
+        "prev": {
+          "href": "https://private-33c60-amm3.apiary-mock.com/liquidity_pools/1/operations?cursor=113725249324879873&limit=10&order=desc"
+        }
+      },
+      "_embedded": {
+        "records": [
+          {
+            "id": "3697472920621057",
+            "paging_token": "3697472920621057",
+            "transaction_successful": true,
+            "source_account": "GBB4JST32UWKOLGYYSCEYBHBCOFL2TGBHDVOMZP462ET4ZRD4ULA7S2L",
+            "type": "liquidity_pool_deposit",
+            "type_i": 22,
+            "created_at": "2021-11-18T03:47:47Z",
+            "transaction_hash": "43ed5ce19190822ec080b67c3ccbab36a56bc34102b1a21d3ee690ed3bc23378",
+            "liquidity_pool_id": "1",
```

Comment:
We should use the correct pool ID here too:
```suggestion
            "liquidity_pool_id": "ae44a51f6191ce24414fbd1326e93ccb0ae656f07fc1e37602b11d0802f74b9a",
```